pegasus/sites.v3/code.org/views/elementary_school_curricula_video.haml

```diff
@@ -1,37 +1,40 @@
-- video_curricula_index = 5
+- show_game_design_curriculum = DCDO.get('curriculum-launch-2024', false)
 
-- video_curricula_tags = [hoc_s(:module_grade_k_5), hoc_s(:module_grade_3_5), hoc_s(:module_grade_3_12), hoc_s(:module_grade_3_12), hoc_s(:module_grade_k_12)]
+- video_curricula_index = 6
 
-- video_curricula_title = [hoc_s(:curriculum_name_csf), hoc_s(:curriculum_name_csc), hoc_s(:curriculum_name_artificial_intelligence), hoc_s(:curriculum_name_maker), hoc_s(:hour_of_code)]
+- video_curricula_tags = [hoc_s(:module_grade_k_5), hoc_s(:module_grade_3_5), hoc_s(:module_grade_3_12), hoc_s(:module_grade_3_12), hoc_s(:module_grade_3_12), hoc_s(:module_grade_k_12)]
 
-- video_curricula_desc = [hoc_s(:csf_page_top_desc), hoc_s(:csc_page_top_desc), hoc_s(:ai_hero_desc), hoc_s(:curriculum_desc_maker), hoc_s(:hoc_homepage_what_is_hoc_subhead)]
+- video_curricula_title = [hoc_s(:curriculum_name_csf), hoc_s(:curriculum_name_csc), hoc_s(:curriculum_name_artificial_intelligence), hoc_s(:curriculum_name_maker), hoc_s('curricula_pages.game_design.title'), hoc_s(:hour_of_code)]
+
+- video_curricula_desc = [hoc_s(:csf_page_top_desc), hoc_s(:csc_page_top_desc), hoc_s(:ai_hero_desc), hoc_s(:curriculum_desc_maker), hoc_s('curricula_pages.game_design.desc'), hoc_s(:hoc_homepage_what_is_hoc_subhead)]

```

Comment:
These green/red highlights are a bit misleading: I just added a new tag, title, and description to each of the three arrays here: `hoc_s(:module_grade_3_12)`, `hoc_s('curricula_pages.game_design.title')`, and `hoc_s('curricula_pages.game_design.desc')` respectively